commands/databases.go

```diff
@@ -599,6 +600,21 @@ To see a list of your databases and their IDs, run `+"`"+`doctl databases list`+
 	return cmd
 }
 
+func databaseInstallUpdate() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:   "install_update",
+			Short: "Display commands for starting installation of updates",
+			Long:  `The ` + "`" + `doctl databases install_update` + "`" + ` commands allow you to starts installation of updates for your databases`,
+		},
+	}
+	cmdDatabaseInstallUpdate := CmdBuilder(cmd, RunDatabaseInstallUpdate, "install_update <database-cluster-id>", "Starts installation of updates ", "Starts installation of updates ", Writer)
```

Comment:
Here you create a new `install_update` sub-group under `databases` and add an `install_update` command to it. So to use this command, you must run:

    doctl databases install_update install_update <database-cluster-id> 

Instead, I think grouping this with the existing maintenance window commands might make a lot of sense. It keeps sub-commands related to applying updates together. So we'd have something like:

```
$ doctl databases maintenance-window -h

# ...

Available Commands:
  get         Retrieve details about a database cluster's maintenance windows
  update      Update the maintenance window for a database cluster
  install     Start installation of updates for a database cluster outside of a maintenance window
```

If we attach it to the top-level `databases` command, please use `install-update` with a hyphen not an underscore for consistency with other commands.